src/components/atoms/AppInput.vue

```diff
@@ -0,0 +1,47 @@
+<script>
+export default {
+  name: "AppInput",
+  props: {
+    modelValue: {
+      type: String,
+      required: true,
+    },
+    placeholder: {
+      type: String,
+      required: true,
+    },
+  },
+  methods: {
+    handleEnter() {
+      this.$emit("enter");
```

Comment:
Just small note. It is recommended to declare emits - https://vuejs.org/guide/components/events.html#declaring-emitted-events